Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -68,6 +68,13 @@ protected override void Dispose(bool disposing)
 				if (disposing)
 				{
 					_itemsSource?.Dispose();
+					_itemsSource = null;
+
+					//clear data
+					CollectionView.DataSource = null;
+					CollectionView.ReloadData();
```

Comment:
Perhaps after it is worth adding `CollectionView.CollectionViewLayout.InvalidateLayout();` Like as in `UpdateItemsSourceAndReload()` function.

Reply:
Thanks, added.